### PR TITLE
New package: perl-Stow

### PR DESCRIPTION
**Pull request by harrigan**

### CODE REVIEW

File: srcpkgs/perl-Stow/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+wrksrc="Stow-v${version}"
+build_style=perl-ModuleBuild
+hostmakedepends="perl perl-Module-Build"
+makedepends="${hostmakedepends}"
+depends="${makedepends}"
+short_desc="Perl module for managing symlinks"
+maintainer="Martin Harrigan <martinharrigan@gmail.com>"
+homepage="http://search.cpan.org/dist/Stow"
+license="Artistic, GPL-1"
+distfiles="http://search.cpan.org/CPAN/authors/id/A/AS/ASPIERS/Stow-v${version}.tar.gz"
```
Comment (the-maldridge) [inline srcpkgs/perl-Stow/template]: I'd be much happier merging this if you used CPAN_SITE here.

**Comment by the-maldridge** [comment]
I'm reasonably sure this should be a noarch package as well.

**Comment by harrigan** [comment]
I changed the distfiles to `${CPAN_SITE}/Finance/ASPIERS/Stow-v${version}.tar.gz` even though it is an odd location -- the module has nothing to do with finance. Also, I intend to follow this up with a package for Gnu Stow which wraps the above in a CLI.

**Comment by the-maldridge** [comment]
Looks good but this should be only one commit, please rebase and squash.

**Pull request closed without merging**